scss/_base_code.scss

```diff
@@ -45,6 +45,7 @@
     border-radius: $border-radius;
     color: $color-dark;
     display: block;
+    font-family: unquote($font-monospace);
```

Comment:
We should check if Ubuntu is in the sites font stack before setting this.

Reply:
How can we check?

Reply:
I just copied this from the code-copyable component, this is not a new addition.

Reply:
Sorry I was wrong yesterday, this is correct.

Reply:
Can you add `pre` to the font-family set block above? 

If we are setting the font-family to mono then shouldn't it inherit the other aspects the font such as:
```css
font-family: unquote($font-monospace);
font-weight: 300;
text-align: left;
```